src/main/java/com/minecolonies/coremod/blocks/BlockSolidSubstitution.java

```diff
@@ -63,7 +66,7 @@ private void initBlock()
      * @return true
      */
     //todo: remove once we no longer need to support this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)